CodeForPittsburgh/law-reading-robot-data
# Create primary migration script and README for Supabase Dev Environment setup.

## What I did

Exactly what it says on the title! The README includes instructions for how to set up the Supabase Dev Environment, as well as initial creation instructions for admins.

## Issue #s Addressed

Trello Task [Supabase dev environment for back end](https://trello.com/c/kqg0J6Ei/38-supabase-dev-environment-for-back-end)

## How to test it

- Follow the instructions given and see if you are able to properly set up the Supabase Dev environment. Tables appearing should match what is currently in the main Supabase database. 
- Run rss_etl.py. If successful, your local dev environment database should populate with six rows in both the "Bills" and "Revisions" tables.
